### DISCUSSION TITLE
关于抢票需要填写完整手机号的问题

### DISCUSSION THREAD

**Discussion by joukjess14** [mikumifa/biliTickerBuy#385] [提问专区]
马上第二轮抢票，今天看到群里有说可能会出现填写完整手机号的过程
![e13b18e2b39305834c9b42b1e692ab1d](https://github.com/mikumifa/biliTickerBuy/assets/25680827/52c88468-ae86-4476-914a-a729cc7f45c1)
这个在用抢票软件的时候，会不会出现这种情况，出现了怎么处理呢

**Comment by flipwind** [comment]
![03f17c684413d02c233d775a79b257c8](https://github.com/mikumifa/biliTickerBuy/assets/89458091/00ad35ce-bcdf-43b9-834f-04f877e69a32)

**Comment by SouthSeafireworks** [comment]
好像没测试过，手抢防着点

**Comment by joukjess14** [comment]
您好，想问一下发了啥，现在404看不到了

在 2024-07-06 11:06:00，FlipWind ***@***.***> 写道：


03f17c684413d02c233d775a79b257c8.png (view on web)
